packages/hop-node/src/db/TransferRootsDb.ts

```diff
@@ -390,11 +485,12 @@ class TransferRootsDb extends TimestampedKeysDb<TransferRoot> {
       }
 
       return (
-        item.bondTransferRootId &&
+        item.transferRootId &&
         item.transferRootHash &&
+        !item.committed &&
+        item.totalAmount &&
         item.bonded &&
         item.destinationChainId &&
```

Comment:
`item.destinationChainId` needs to be removed from this db getter since it won't always be available.

(Sorry, I misspoke earlier and should have called this out when I OKed 12f3b0d61a843e8e8eb07335a13cde0f826929ca)

Reply:
Removed in https://github.com/hop-protocol/hop/pull/318/commits/48bcdaf9078daa59da9e74ca6af291aec5d1489c